src/Sulu/Bundle/AdminBundle/Resources/js/components/FolderList/Folder.js

```diff
@@ -0,0 +1,51 @@
+// @flow
+import React from 'react';
+import Icon from '../Icon';
+import folderStyles from './folder.scss';
+
+const FOLDER_ICON = 'folder-o';
+
+type Props = {
+    /** The id which will be passed as an argument inside the onClick handler */
+    id: string | number,
+    /** The subtext underneath the title */
+    meta: string,
```

Comment:
Hm... `meta` is a bit a strange name... Doesn't really say anything. Maybe `description`? Or `info`?